### PR TITLE
Have getSenders()/getReceivers() return only non-stopped ones.

### PR DESCRIPTION
**Pull request by jan-ivar**
Fix for https://github.com/w3c/webrtc-pc/issues/1975.

***
<a href="https://pr-preview.s3.amazonaws.com/jan-ivar/webrtc-pc/pull/1977.html" title="Last updated on Sep 7, 2018, 7:06 PM GMT (950cc15)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/1977/e19d64f...jan-ivar:950cc15.html" title="Last updated on Sep 7, 2018, 7:06 PM GMT (950cc15)">Diff</a>

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -4855,51 +4855,46 @@ <h2>Methods</h2>
             <dt><code>getSenders</code></dt>
             <dd>
               <p>Returns a sequence of <code><a>RTCRtpSender</a></code> objects
-              representing the RTP senders that are currently attached to this
-              <code><a>RTCPeerConnection</a></code> object.</p>
-              <p>The <dfn id=
+              representing the RTP senders that are part of non-stopped
```
Comment (fippo) [inline webrtc.html]: nit: belong to?

File: webrtc.html [hunk 2]
```diff
@@ -4855,51 +4855,46 @@ <h2>Methods</h2>
...
                 <li>Let <var>senders</var> be a new empty sequence.</li>
                 <li>For each <var>transceiver</var> in <var>transceivers</var>,
                   <ol>
-                    <li>Let <var>sender</var> be
-                    <var>transceiver</var>'s <a>[[\Sender]]</a>.</li>
-                    <li>Add <var>sender</var> to <var>senders</var>.</li>
+                    <li>If <var>transceiver</var>.<a>[[\Stopped]]</a> is
+                      <code>false</code> add
+                      <var>transceiver</var>.<a>[[\Sender]]</a> to
+                      <var>receivers</var>.</li>
```
Comment (fippo) [inline webrtc.html]: :spaghetti:, this should be senders instead of receivers.

File: webrtc.html [hunk 3]
```diff
@@ -4855,51 +4855,46 @@ <h2>Methods</h2>
...
-              <p>The <dfn id=
+              representing the RTP senders that belong to non-stopped
+              <code><a>RTCRtpTransceiver</a></code> objects currently attached
+              to this <code><a>RTCPeerConnection</a></code> object.</p>
+              <p>When the <dfn id=
               "dom-peerconnection-getsenders"><code>getSenders</code></dfn>
-              method MUST return the result of executing the
-              <code><a>CollectSenders</a></code> algorithm.</p>
-              <p></p>
-              <p>We define the <dfn>CollectSenders</dfn> algorithm as
```
Comment (henbos) [inline webrtc.html]: The CollectSenders algorithm is still referenced from addTrack, now it is no longer defined. Update that step to get the result of getSenders()?

**Comment by jan-ivar** [comment]
@henbos Thanks for spotting that. I brought back the *CollectSenders* algorithm, now with non-stopped senders only.

A net benefit: the following now works:
```js
const transceiver = pc.addTrack(track);
transceiver.stop();
pc.addTrack(track); // No longer throws InvalidAccessError

```

**Pull request merged**